src/t_zset.c

```diff
@@ -247,8 +269,7 @@ static zskiplistNode *zslInsertNode(zskiplist *zsl, zskiplistNode *node) {
 }
 
 /* Insert a new node in the skiplist. Assumes the element does not already
- * exist (up to the caller to enforce that). The skiplist takes ownership
- * of the passed SDS string 'ele'. */
+ * exist (up to the caller to enforce that). */
```

Comment:
(regarding zslInsert) Mention that the element is copied?

```suggestion
 * exist (up to the caller to enforce that). The string 'ele' is copied. */
```